### ISSUE TITLE
Change loss accumulation

### ISSUE THREAD

**Issue by jbirkesteen** [RasmusBrostroem/ConnectFourRL#28]
Try using `torch.sum` instead of `torch.mean` when accumulating the loss for each `batch_size`.
